docker/rofl-dev/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ghcr.io/oasisprotocol/oasis-core-dev:stable-24.3.x AS oasis-core-dev
+
```

Comment:
See https://github.com/oasisprotocol/oasis-sdk/pull/2063#issuecomment-2517387657
```suggestion

ENV RUSTFLAGS="-C target-feature=+aes,+ssse3"
```

Reply:
^^ Please test if this works. (remove .cargo folder and see if ROFL still compiles)